connector-sdk/core/src/main/java/io/camunda/connector/api/inbound/webhook/VerifiableWebhook.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.api.inbound.webhook;
+
+import java.util.Map;
+import java.util.function.Function;
+
+public interface VerifiableWebhook {
+  default WebhookHttpVerificationResult verify(WebhookProcessingPayload payload) throws Exception {
+    return null;
+  }
+
+  default Function<Map<String, Object>, WebhookHttpVerificationResult> verificationExpression() {
+    return response -> null;
+  }
```

Comment:
Is this method necessary?

Reply:
Why not just execute `verify` and check the result?

Reply:
Removed